docs/public/specifications/tests/README.md

```diff
@@ -132,6 +132,7 @@ project. All the details are in the [Master Test Plan].
 |[NAMESERVER12](Nameserver-TP/nameserver12.md)|Test for unknown EDNS flags|
 |[NAMESERVER13](Nameserver-TP/nameserver13.md)|Test for truncated response on EDNS query|
 |[NAMESERVER14](Nameserver-TP/nameserver14.md)|Test for unknown version with unknown OPTION-CODE|
+|[NAMESERVER15](Nameserver-TP/nameserver15.md)|Checking for software version|
```

Comment:
This file is to be updated at release time so it is not needed to include the change now.